packages/components/src/text/text.tsx

```diff
@@ -12,7 +12,7 @@ type Weight = NonNullable<Variants['weight']>
 
 type Props = {
   children: React.ReactNode
-  color?: ColorTokens
+  color?: ColorTokens | string
```

Comment:
In order to simplify the implementation of custom colors (which is an edge case), we should force the consumer to provide an object instead of trying to handle every possible color space. 

(are you planning to update the code to handle [OKLCH](https://caniuse.com/?search=OKLCH) color space? 😉)

Therefore do this:
```ts
color?: ColorTokens | { r: string, g: string, b: string }
```

... and change opacity using the `alpha` channel.

Reply:
It is also much more straightforward to transform HEX to RGB.

Reply:
Or force consumers to pass only HEX using TypeScript template literal types.

```ts
color?: ColorTokens | `#${string}`
```